src/hexen/h2_main.c

```diff
@@ -357,6 +358,81 @@ void D_SetGameDescription(void)
     }
 }
 
+static struct
```

Comment:
This could be `const`

Reply:
That's a good suggestion. Should it maybe be done in a separate commit for Doom, Hexen and Strife altogether (after accepting this patch)? Because the Hexen code in question was based on Doom's, and I see that the const qualifier is still not used in any of Doom and Strife's definitions of the gameversions struct (short of strings).

I can also add it for Hexen only as a separate commit to this PR, so this isn't forgotten for Doom and Strife later on.

Reply:
Either way is ok with me.